crates/sui/src/keytool.rs

```diff
@@ -54,8 +54,14 @@ impl KeyToolCommand {
             }
 
             KeyToolCommand::Show { file } => {
-                let keypair: AuthorityKeyPair = read_keypair_from_file(file)?;
-                println!("Public Key: {}", encode_bytes_hex(keypair.public()));
+                let p = file.clone();
+                let res: Result<AuthorityKeyPair, anyhow::Error> = read_keypair_from_file(file);
```

Comment:
we don't need to clone the file if we pass in the file by ref

```suggestion
                let res: Result<AuthorityKeyPair, anyhow::Error> = read_keypair_from_file(&file);
```